src/test/java/io/vertx/core/net/impl/HostAndPortTest.java

```diff
@@ -1,5 +1,6 @@
 package io.vertx.core.net.impl;
 
+import io.vertx.core.http.HttpTestBase;
```

Comment:
those tests don't use a server, so they should use literal values instead, that makes them less readable